src/styles/blocks/card.scss

```
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card--width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  color: $card-color-text;
+
+  margin-right: $marginX;
+  margin-bottom: $marginY;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: block;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__description {
+    line-height: 18px;
+    margin: 0 0 4px;
+    font-weight: $font--weight--medium;
+    font-size: 12px;
+    color: $card-color-text;
+  }
+
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
+    line-height: 14px;
+
+    font-size: 10px;
+    color: $card-color--text-light;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-text {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-color-text;
+    padding-bottom: 1px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      color: $card-color--text-light;
+
+      margin: 0;
+    }
```

These are two different elements.
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    line-height: 18px;
    margin-bottom: 16px;
  }

  &__price-label {
    font-size: 12px;
    color: $card-color--text-light;

    margin: 0;
  }
```